### ISSUE TITLE
Doubt in the AFT loss computation

### ISSUE THREAD

**Issue by BheeshmSharma** [amazon-science/adaptive-feature-transfer#1]
In prior.py >
Line 117-119
# print first 5 x 5 block
self.k = k[:5, :5].detach().cpu()
self.k_target = k_target[:5, :5].detach().cpu()

Why only 5 samples in kernel based computation has been used here?  Please clarify.

Thanks

**Comment by shikaiqiu** [comment]
Hi Bheeshm,

The code you highlighted is only used for printing and debugging. The loss is computed using the full kernel.

Best,
Shikai
